### PR TITLE
[Reproduction] Reproduce training results of RegnetY

### PR DESCRIPTION
**Pull request by Ezra-Yu**
## Motivation

Add RegnetY

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: configs/_base_/datasets/imagenet_bs128.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# dataset settings
+dataset_type = 'ImageNet'
+
+img_norm_cfg = dict(
+    # The mean and std are used in PyCls when training and testing RegNets
```
Comment (ycxioooong) [inline configs/_base_/datasets/imagenet_bs128.py:5]: we may write the comment in a more general way. e.g.
```
The mean and std used in PyCls for training and testing models with reversed color channels with mmcls default setting,
for instance, RegNets. 
```

File: configs/_base_/datasets/imagenet_bs128.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    mean=[103.53, 116.28, 123.675],
```
Comment (ycxioooong) [inline configs/_base_/datasets/imagenet_bs128.py:6]: also, we may consider following our own setting when we are able to align the performance using our own models.

**Comment by ycxioooong** [comment]
Do we need to unify SE and SE Layer here in this PR? @Ezra-Yu

**Comment by Ezra-Yu** [comment]
Because the commits are too messy, another one will be reopened later.

**Pull request closed without merging**